README.md

```diff
@@ -88,7 +88,7 @@ Install the following dev dependencies:
 ```sh
 npm install --save-dev @types/node @types/react @types/react-dom typescript
 npm install --save-dev astro react react-dom
-tsc --init -y
+npx tsc --init
```

Comment:
I didn't have `tsc` installed globally. This was the most straightforward way  to make it work. (And the `-y` option didn't work on my system.)

**However:** I'd recommend skipping this step! All it does is create a `tsconfig.json` file, with settings that are insufficient for us. (E.g. no JSX support, etc.) We already have a step that instructs the user to create a `tsconfig.json` with proper settings, so we should be fine without this step.